production-antora-playbook.yml

```
@@ -70,6 +70,9 @@ content:
   - url: https://github.com/couchbaselabs/userprofile-couchbase-mobile.git
     branches: [standalone, query, sync, backgroundfetch]
     start_path: content
+  - url: https://github.com/couchbaselabs/userprofile-couchbase-mobile-xamarin.git
+    branches: [standalone, query, sync, backgroundfetch]
```

```suggestion
    branches: standalone
```

The build system will fail if it can't find the content on specified branches (from what I can see the `query`, `sync` and `backgroundfetch` branches show a single .gitignore file currently).